sematic/plugins/building/docker_builder.py

```diff
@@ -389,6 +476,21 @@ def _find_build_config_files(script_path: str) -> List[str]:
     return [file_candidate] if os.path.isfile(file_candidate) else []
 
 
+def _get_docker_client(
```

Comment:
Nit. More of a factory than a getter.
```suggestion
def _make_docker_client(
```